modules/tinymce/changelog.txt

```diff
@@ -1,5 +1,6 @@
 Version 5.4.0 (TBD)
     Changed `mceInsertTable` command and `insertTable` API method to take optional header rows and columns arguments #TINY-6012
+		Changed the code that filters commands out of the `undo` stack to be case insensitive.
```

Comment:
There's a missing space here and this should probably be `Fixed` as this was unintended behaviour and not really changing anything from a users point of view.

- [ ] Fix changelog

Reply:
New commit should fix the changelog.